app/controllers/locations_controller.rb

```diff
@@ -0,0 +1,19 @@
+class LocationsController < ApplicationController
+  def update
+    @location = find_location
+
+    @location.update!(location_params)
+
+    redirect_to profile_url, flash: { success: t(".success") }
```

Comment:
@geoffharcourt for your information: you'd need to wrap the `success` key in a `flash` hash